packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/workspace-member.object-metadata.ts

```diff
@@ -60,6 +60,7 @@ export class WorkspaceMemberObjectMetadata extends BaseObjectMetadata {
     description: 'Workspace member avatar',
     icon: 'IconFileUpload',
   })
+  @IsNullable()
```

Comment:
@magrinj it should not be nullable, it should have a default value ''

Reply:
actually this should already be implemented as '' is the default default value of text field type